src/pages/Positions/Positions.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Positions.scss';
+import Nav from '../../components/Nav.jsx';
+import Card from '../../components/Card.jsx';
+import PositionBtn from '../Positions/PositionBtn.jsx';
+
+export default function Positions() {
+  const positionJobBtn = {
+    jobGroup: [
+      '전체',
+      '서버/백엔드 개발자',
+      '프론트엔드 개발자',
+      '웹 풀스택 개발자',
+      '안드로이드 개발자',
+      'IOS 개발자',
+      '클스플랫폼 웹개발자',
+      '게임 클라이언트 개발자',
+      '게임 서버 개발자',
+    ],
+  };
```

Comment:
- 컴포넌트가 렌더링될 때마다 다시 그릴 필요가 없는 상수 데이터이기 때문에, 컴포넌트 외부에서 선언하셔도 됩니다.
- positionJobBtn은 객체이고 하나의 프로퍼티밖에 없는데, 새로운 프로퍼티가 추가될 예정인가요? 그렇지 않다면 JOB_GROUP_LIST라는 배열로 관리하면 될 것 같습니다. 그리고 나서 각각의 요소를 단순 string이 아닌 객체로 관리하는 게 추후에 id나 백엔드에 전달할 값을 관리하기 좋을 것 같네요!
```suggestion
  const JOB_GROUP_LIST = [{ id: 1, group: '전체'}, ...];
```

Reply:
아 이건 테스트용으로 만들어본거라서... 지우고 다르게 만들어놓은게있습니다 다시 올리겠습니다